var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -156,6 +156,13 @@ def libs(self):
             libraries, root=self.prefix, shared=shared, recursive=True
         )
 
+    variant('context-impl',
+            default='fcontext',
+            values=('fcontext','ucontext','winfib'),
+            multi=False,
+            description='Use the specified backend for boost-context',
+            when='+context')
```

Comment:
What version of Boost introduced this feature? I'm working to make sure variants are correctly mapped to versions where they are actually available.

Reply:
I'll run my automated tests and check.

Reply:
Incidentally there's an option for supporting valgrind that would be great to get rolled in as well:

https://www.boost.org/doc/libs/1_79_0/libs/context/doc/html/context/stack/valgrind.html

Reply:
Oh, very nice! I will try to see about getting at least a reference list of the features/properties.